include/zephyr/drivers/aesc/ip_identification.h

```diff
@@ -0,0 +1,100 @@
+/** @file
```

Comment:
This seems too vendor-specific to be placed here, should this be in `soc/aesc/...` and exposed via `zephyr_include_directories(.)` instead?

Reply:
good idea! I was not very happy about the location either.